sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2304,7 +2304,15 @@ object functions {
    * @group string_funcs
    * @since 1.5.0
    */
-  def ltrim(e: Column): Column = withExpr {StringTrimLeft(e.expr) }
+  def ltrim(e: Column): Column = withExpr {StringTrimLeft(e.expr)}
```

Comment:
You need to update the styles of all the newly added functions in this file. For example,
```Scala
def ltrim(e: Column): Column = withExpr {
  StringTrimLeft(e.expr)
}
```